<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Sum_btn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="Sum_result">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_A">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_B">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Sum_btn</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>sum()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sum()</slot>
 </slots>
</ui>
